### PR TITLE
Wizard Of Os Front-End Theme

### PR DESCRIPTION
**Pull request by misterzik**
Enjoy a cleaner UI with Wizard Of Os + Web-term

### CODE REVIEW

**Comment by IonicaBizau** [comment]
Hey @misterzik,

Thanks for contributing. Please add yourself as contributor in `package.json` and I will leave some inline comments. :star2: 

It looks like you have replaced the things and some of them should be rolled back. You can keep pushing into this branch and this PR will be automatically updated.

File: bin/public/css/style.css [hunk 1]
```diff
@@ -1,28 +0,0 @@
-div {
```
Comment (IonicaBizau) [inline bin/public/css/style.css]: This is needed for the default terminal. So, it should be brought back.

File: bin/public/horizontal-split.html [hunk 2]
```diff
@@ -1,24 +0,0 @@
-<!DOCTYPE html>
```
Comment (IonicaBizau) [inline bin/public/horizontal-split.html]: This should *not* be deleted. It's the horizontal split introduced in latest version.

File: bin/public/index.html [hunk 3]
```diff
@@ -1,17 +1,111 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
```
Comment (IonicaBizau) [inline bin/public/index.html]: Is it possible to separate these files (Wizard of OS) into another directory? I can take care of loading them if some special option is provided.

File: bin/public/css/style.css (hunk 1 above)
Comment (misterzik) [inline bin/public/css/style.css]: I kept those files on the other stylesheet, as well. I though keeping same styles in both will only make waiting/loading time longer, but i understand your point.

File: bin/public/horizontal-split.html (hunk 2 above)
Comment (misterzik) [inline bin/public/horizontal-split.html]: I added a horizontal look as well, Please head to localhost/doublerun and for one view localhost/run.
I was wondering if your router had something similar to angular ngview?

File: bin/public/index.html (hunk 3 above)
Comment (misterzik) [inline bin/public/index.html]: Yes, indeed. Should i create a different folder inside public? perhaps 

Themes
-> Layout 1
-> Layout 2

I've one more layout, that im still working on.

Comment (IonicaBizau) [inline bin/public/index.html]: Yes, that would clean up the things a little bit. :smile:

Comment (misterzik) [inline bin/public/index.html]: how would you pick which theme to use tho?

Comment (IonicaBizau) [inline bin/public/index.html]: Just by opening `localhost:<port>/wizard-of-os/index.html`?

**Comment by misterzik** [comment]
Are this files better for merge? @IonicaBizau

**Comment by IonicaBizau** [comment]
@misterzik Oh, looks good! :sparkles: Will test it this evening and will fix little things I will see or I will ping you if I find something bigger. :smile: Looks like it's a nice work done there. :clap:

**Comment by IonicaBizau** [comment]
Oh, totally forgot about this!!! Sorry. Will check it soon.

**Comment by IonicaBizau** [comment]
I just tested it and pushed some fixes. Really nice work here! :sparkles:

**Comment by IonicaBizau** [comment]
:ship: **`4.0.0`** :tada:

**Comment by IonicaBizau** [comment]
Thanks for the nice work! :cake:

**Pull request merged**